### ISSUE TITLE
KF 1.7.0 central dashboard LDAP log out stuck at white screen

### ISSUE THREAD

**Issue by yingding** [kubeflow/manifests#2431]
While log out from the central dashboard web app of KF 1.7.0 manifests on my on-prem microk8s V1.24.12, I stuck with a white screen with url `https://<domain>/authservice/site/after_logout`

**Comment by annajung** [comment]
This is a known issue https://github.com/kubeflow/kubeflow/issues/5704, further discussion should be moved to notebook WG.

**Comment by shan100github** [comment]
Facing the same issue as mentioned by @yingding 

Adding: in 1.6.x didn't face any issues and was redirected back to the login page successfully. 
Auth handled by dex FYI.

**Comment by yingding** [comment]
The pr #2461 fixed my issue.

A workaround is to apply the fix from PR #2461, reinstall the oidc-authsevice with kustomize 5.0.1
```
while ! kustomize build example | kubectl apply -f -; do echo "Retrying to apply resources"; sleep 10; done
```
and restart the pods
```
kubectl -n auth delete pod dex-64775f9cf6-6tv4z
kubectl -n kubeflow delete pod centraldashboard-b8f749ccd-sk9sm
kubectl -n istio-system delete pod authservice-0
```
@annajung Can you help review the PR #2461 ?

**Comment by yingding** [comment]
I am closing this issue, since the fix is merged.
